server/accounts.go

```diff
@@ -4114,18 +4120,64 @@ type transform struct {
 	src, dest string
 	dtoks     []string
 	stoks     []string
-	dtpi      []int8
+	dtpi      [][]int // destination token position indexes
+	dtpinp    []int32 // destination token position index number of partitions
+}
+
+func getMappingFunctionArgs(functionRegEx *regexp.Regexp, token string) ([]string, error) {
+	commandStrings := functionRegEx.FindStringSubmatch(token)
+	if len(commandStrings) > 1 {
+		args := commaSeparatorRegEx.Split(commandStrings[1], -1)
+		return args, nil
```

Comment:
return commaSeparatorRegEx.Split(), nil ?

Reply:
Thanks forgot to clean that up as part of how I wrote different iterations of the code, probably didn't change it because the compiler takes care of optimizing that anyways but it doesn't add anything to source code readability so should and will be cleaned up.